README.md

```diff
@@ -392,6 +392,8 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [r2c](https://jobs.lever.co/returntocorp/82c10097-a669-417a-9434-66a98a8bca1f) | San Francisco, CA; or Remote | Software Engineer - Internship, Application Development |
 | [Excelitas](https://www.excelitas.com/job/2022-10/software-engineering-intern) | Billerica, MA | Software Engineering Intern (No Visa Sponsorship) |
 | [Teradata](https://careers.teradata.com/jobs/215587/cloud-software-engineering-intern) | Remote | Cloud Software Engineering Intern |
+| [klue](https://jobs.lever.co/klue/b88e45c3-c067-410b-a7d3-ca23142b9bef/apply?lever-source=LinkedIn) | Vancouver, BC | [Data Analyst Intern](https://jobs.lever.co/klue/b88e45c3-c067-410b-a7d3-ca23142b9bef/apply?lever-source=LinkedIn) |
```

Comment:
I recommend sending the applicant to the job page description (`https://jobs.lever.co/klue/b88e45c3-c067-410b-a7d3-ca23142b9bef`) instead of straight to the application page. That way, they can read more about the position.